rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -1664,9 +1667,13 @@ public void visitMethodCallExpression(MethodCallExpression call) {
                     Expression selectExpr = visit(call.getObjectExpression());
```

Comment:
I did some change below this line, but I have the feeling I am at the wrong track. For instance with following groovy code:

```groovy
(map.get("foo")).equals("bar")`
```

I can retrieve here an expression that looks printed like `(map.get("foo")`. I think you would expect it evenly distributed like `(map.get("foo"))`. So maybe that's what should be changed instead of the code here below.

Reply:
I stop working on this for now. I can't wrap my head around how the parentheses are set. I probably need a little bit more experience before I start working on parser code 😇.